recipes/imbalanced-learn/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "imbalanced-learn" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "5dfd6773b5f5e067ebcc6b032907ce721227a2f2eb0b684f3b169c65a10b5257" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy 1.9.*  # [not (win and py36)]
+    - numpy 1.11.*  # [win and py36]
```

Comment:
Is it linking to `numpy` or does it merely require it?

Reply:
we only require it.

Reply:
Ok. So these pinnings are based on the idea of linking to `numpy`. If you are only requiring it, would just use the same version constraints that you are using in `setup.py` for `build` and `run` here.

Reply:
@glemaitre Then you don't need to do the version pinning; just a runtime plain `numpy` (or maybe `numpy >=1.10` if you need some feature from 1.10 or whatever) should be enough. That's only if you're compiling against numpy libraries.

Reply:
@dougalsutherland The reason of specifying at build is due to our setup: https://github.com/conda-forge/staged-recipes/pull/4146#discussion_r143770588